tests/test_more.py

```diff
@@ -5395,59 +5396,59 @@ def test_key_partially_hashable(self):
         )
 
     def test_vs_unique_everseen(self):
-        input = 'AAAABBBBCCDAABBB'
-        output = [e for e, j, u in mi.classify_unique(input) if u]
+        input_ = 'AAAABBBBCCDAABBB'
```

Comment:
No.  The advice to not shadow builtins is mainly applicable in the global namespace where it is very easy to create confusion.  For local variable names, the advice to not shadow a builtin is only applicable when if the code is easily misread. In this case, it is very clear that `input` is input data and not the `input` builtin function.

Note, `input_` is harder to read. It always slows down someone reading the code.

IMO, this edit makes the code worse.